care/facility/models/asset.py

```diff
@@ -17,6 +17,13 @@ def get_random_asset_id():
     return str(uuid.uuid4())
 
 
+class AvailabilityStatus(models.IntegerChoices):
```

Comment:
Since this is going to be used in metabase it might make sense to switch to string based values to make it easier for the metabase folks